### DISCUSSION TITLE
(101) tgbot-cpp library can't parse json response. <unspecified file>(2): invalid code sequence

### DISCUSSION THREAD

**Discussion by MrAndreyka** [reo7sp/tgbot-cpp#309] [General]
When clicking on the inline button in the bot, it generates an error. It doesn't reach the handler.

but->callbackData = "І±data±:І±task±:1,±user±:2і,±name±:±text±,±type±:±edit±і";

If Telegram lets messages through, then the bot shouldn't crash either

**Comment by reo7sp** [comment]
The Boost.PropertyTree parser involved here was removed in v1.12 and replaced with nlohmann/json. Update to v1.12.3. `callback_data` must still be valid UTF-8 and at most 64 bytes; encode binary or arbitrary structured data as base64url or another ASCII format.
